### PR TITLE
Fix command indentation

### PR DESCRIPTION
**Pull request by vladklokun**
## Describe your changes

This PR contains the following changes to be merged into `dev`:
- introduces the initial implementation of the `kubescape fix` command developed in #898
- adds a fix that aims to keep the formatting of the original file when fixing it, as developed in #959 

## This PR fixes:

* Resolved #

## Checklist before requesting a review

- [x] My code follows the style guidelines of this project
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] I have performed a self-review of my code
- [x] If it is a core feature, I have added thorough tests.
- [x] New and existing unit tests pass locally with my changes

**Please open the PR against the `dev` branch (Unless the PR contains only documentation changes)**

### CODE REVIEW

**Comment by vladklokun** [comment]
This breaks on Windows because it uses `\r\n` as end-of-line characters, while we use `\n` to write fixes.

**Comment by suhasgumma** [comment]
Hi @vladklokun, be mindful of this issue https://github.com/kubescape/regolibrary/issues/186

This will cause errors while deploying fixed Kubernetes manifest files of workloads.

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1 @@
+core/pkg/fixhandler/testdata/inserts/*-crlf-newlines.yaml text eol=crlf
```
Comment (dwertent) [inline .gitattributes:1]: [Question]: Does this mean we support only CRLF line endings? If not, why is it necessary to explicit this attribute?

File: core/pkg/fixhandler/fixhandler.go [hunk 2]
```diff
@@ -0,0 +1,361 @@
...
+
+		rfi := ResourceFixInfo{
+			FilePath:        absolutePath,
+			Resource:        resourceObj,
+			YamlExpressions: make(map[string]*armotypes.FixPath, 0),
+			DocumentIndex:   documentIndex,
+		}
+
+		for i := range result.AssociatedControls {
+			if result.AssociatedControls[i].GetStatus(nil).IsFailed() {
```
Comment (dwertent) [inline core/pkg/fixhandler/fixhandler.go:156]:
```suggestion
			if !result.AssociatedControls[i].GetStatus(nil).IsPassed() {
```
@Moshe-Rappaport-CA Please make sure to change this as part of the status refactoring.

File: core/pkg/fixhandler/fixhandler.go [hunk 3]
```diff
@@ -0,0 +1,361 @@
...
+		sb.WriteString(fmt.Sprintf("Resource: %s\n", resourceFixInfo.Resource.GetName()))
+		sb.WriteString(fmt.Sprintf("Kind: %s\n", resourceFixInfo.Resource.GetKind()))
+		sb.WriteString("Changes:\n")
+
+		i := 1
+		for _, fixPath := range resourceFixInfo.YamlExpressions {
+			sb.WriteString(fmt.Sprintf("\t%d) %s = %s\n", i, (*fixPath).Path, (*fixPath).Value))
+			i++
+		}
+		sb.WriteString("\n------\n")
```
Comment (dwertent) [inline core/pkg/fixhandler/fixhandler.go:184]: Ok.

**Review by dwertent — APPROVED** [review]
Well done.
I appreciate the efforts that where pt in the tests coverage

File: core/pkg/fixhandler/fixhandler.go (hunk 2 above)
Comment (dwertent) [reply core/pkg/fixhandler/fixhandler.go:156]: @vladklokun This should remain as it is now.

File: core/pkg/fixhandler/fixhandler.go (hunk 3 above)
Comment (vladklokun) [reply core/pkg/fixhandler/fixhandler.go:184]: Sorry, I don’t get it. Am I missing something?

**Comment by vladklokun** [comment]
> Hi @vladklokun, be mindful of this issue [kubescape/regolibrary#186](https://github.com/kubescape/regolibrary/issues/186)
> 
> This will cause errors while deploying fixed Kubernetes manifest files of workloads.

@suhasgumma noted. Thank you, Suhas!

**Review by dwertent — APPROVED** [review]

**Pull request merged**